Add CLI description to the help.sap.com portal (#2370)
kyma-project/cli: docs/user/README.md

# Kyma CLI

## What is Kyma CLI?

The Kyma CLI is a command-line interface tool designed to simplify the use of the Kyma ecosystem. It allows you to manage Kyma modules and applications, deploy simple applications, and more. With Kyma CLI, you can perform complex tasks with simple commands, accelerating development cycles.

> In addition, you can build, push, and deploy, an application to a Kyma cluster with a single command. It automatically detects the Dockerfile in the current directory, builds and pushes the image to the in-cluster registry, and applies the necessary Kubernetes resources.

Reviewer: ```suggestion
In addition, you can build, push, and deploy an application to a Kyma cluster with a single command. It automatically detects the Dockerfile in the current directory, builds and pushes the image to the in-cluster registry, and applies the necessary Kubernetes resources.
```

# Kyma CLI

## What is Kyma CLI?

> The Kyma CLI is a command line interface tool designed to simplify usage of the Kyma ecosystem. It allows developers and administrators to manage Kyma modules and applications, provision clusters, and many more. By using Kyma CLI, users can perform complex tasks with simple commands, accelerating development cycles.

Reviewer 2: Clusters provisioning is something that is right now a part of the CLI but in the future, we plan to get rid of this functionality. I would suggest not mentioning it in docs.

```suggestion
The Kyma CLI is a command line interface tool designed to simplify usage of the Kyma ecosystem. It allows developers and administrators to manage Kyma modules and applications, deploy simple applications, and many more. By using Kyma CLI, users can perform complex tasks with simple commands, accelerating development cycles.
```

With Kyma CLI, you can build, push, and deploy, an application to a Kyma cluster with a single command. It automatically detects the Dockerfile in the current directory, builds and pushes the image to the in-cluster registry, and applies the necessary Kubernetes resources.

Kyma CLI also provides a set of commands to manage Kyma modules efficiently. Users can manage, deploy and configure modules seamlessly. The Kyma CLI module commands allow users to list available and installed modules, enable or disable them, and configure their settings. Modules can be deployed with default or custom configuration. 
## Features
The Kyma CLI provides the following features:

> - Simplified module management
> - Automated deployments
> - Built-in extensibility
> - Integrated service management
> - Creation of Kubeconfigs with OIDC tokens
> - Provisioning Kyma clusters

Reviewer 2: Not all of these items will be a part of the CLI in the future. I would suggest getting rid of them and keeping only those mentioned in [this epic](https://github.com/kyma-project/cli/issues/2232).

```suggestion
- Simplified module management
- Automated deployments
- Built-in extensibility
- Integrated service management
- Commands providing useful automation
```
